site/tutorials/snippets/08/monitoring/setupPrometheus.md

```diff
@@ -2,39 +2,100 @@
 ## Setup Prometheus Monitoring
 Duration: 3:00
 
-After creating a project and service, you can setup Prometheus monitoring and configure scrape jobs using the Keptn CLI. 
+After creating a project and service, you can set up Prometheus monitoring and configure scrape jobs using the Keptn CLI.
 
-To install the *prometheus-service*, execute: 
+Keptn doesn't install or manage Prometheus and its components. Users need to install Prometheus and Prometheus Alert manager as a prerequisite.
+
```

Comment:
Please add a sentence here for the Prometheus installation with helm and the community-managed helm chart that you are using. The purpose of the tutorial is that the user can follow all steps and at the end has a working environment. 
Therefore, also the steps to install Prometheus are needed.

Reply:
Done